### DISCUSSION TITLE
Can I recover the contributions of a lost account?

### DISCUSSION THREAD

**Discussion by aline-doleron** [community/community#81217] [New to GitHub]
Hello,

I had an account with my current email and another email of the company I worked. As I left the company, I've lost the recovery keys, so I was not able to login on my account.
I unlinked my email and created a new github account with it. 

**Question:** Is it possible to recover the contributions (for public repositories) of my lost account?
Previous account: @alinedoleron

Thanks!

**Comment by ananuness** [comment]
Hii Aline!! I'm sorry for what happened, but you can't recover it because all our contributions are linked to the email of the account in question... So if you want those back, you have to use the same email as your actual Primary email (you can see it in `Settings > Emails`).

To prevent it from happening again, you can provide an email (be sure which one you will use) that will be used in any account you may have! Check this out: 

[Setting your commit email address on GitHub](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/setting-your-commit-email-address#setting-your-commit-email-address-on-github)

**Reply by aline-doleron** [reply]
Hello, @ananuness. Thank you so much for kindly answer my question. :tulip: 
I'll try to figure out if I can use the previous email.

Regards!

**Reply by ananuness** [reply]
I'm happy to help! Best regards 🌸

**Reply by aline-doleron** [reply]
Hi @ananuness, just one more question if you can answer yet. I've got my company email and now I'm using it as primary email. 
Do I need to do something more in order to the contributions to be displayed on the timeline? 
For example, my contributions was made to this public repository (https://github.com/liferay/liferay-portal). Do I need to fork this repo on my new account (https://github.com/aline-doleron)?

**Reply by ananuness** [reply]
Hi Aline! So, i was searching more about it and it seems that you have to remove this email from the work account. Sadly, i would recommend you to create a safe email to use from now to don't have this issue anymore if you don't have access to the work account. You can see more about it here in [discussions](https://github.com/orgs/community/discussions/23075)

**Reply by aline-doleron** [reply]
Thanks, Ana! I was reading the discussions you mentioned. Unfortunatelly I think  there's no way to transfer the contributions to my new account. But there's no problem. I'll add a link to the old account. :sweat_smile:  Thank you very much for your attention.
